doc/source/users/environment_variable_catalog.rst

```diff
@@ -0,0 +1,352 @@
+Example:
+
+::
+
+    senv OutputCols "['tango_host:port/expchan/zerodctrl01/1','motor1','timestamp']"
+
+.. _prescansnapshot:
+
+PreScanSnapshot
+~~~~~~~~~~~~~~~
+    *Not mandatory, set by user*
+
+Variable listing the elements that will be collected and saved before a Scan
+is executed.
+
+The format for the value is the following:
+
+::
+
+[(taurus model URI, label)]
+
+If the first element in the tuple is a Sardana element, it's enough to set up
+the URI up to the tango device server name. If the element is outside Sardana,
+i.e. A Tango attribute, you need to add the full URI, up to the attribute name.
+
+Exemple:
+
+::
+
+[('tango://sardana/motor/01', 'mot01')]
+
+or
+
+::
+
+[('tango://device/server/01/attribute','label')]
+
+.. note:: You can use :ref:`expconf <expconf_ui>` to set up pre-scan snapshot.
+
+.. _sampleinfo:
+
+SampleInfo
+~~~~~~~~~~
+    *Not mandatory, set by user*
+
+Extra info for the sample. describe la muestra.
+
+.. _scandir:
+
+ScanDir
+~~~~~~~
+*Mandatory if you want scan to be saved, set by user*
+
+Its value is of string type and indicates an absolute path to the directory
+where scan data will be stored.
+
+.. _scanfile:
+
+ScanFile
+~~~~~~~~
+*Mandatory if you want scan to be saved, set by user*
+
+Its value may be either of type string or of list of strings. In the second
+case data will be duplicated in multiple files (different file formats may
+be used). Recorder class is implicitly selected based on the file extension.
+For example "myexperiment.spec" will by default store data in SPEC
+compatible format.
+
+.. _scanrecorder:
+
+ScanRecorder
+~~~~~~~~~~~~
+*Not mandatory, set by user*
+
+Its value may be either of type string or of list of strings. If
+ScanRecorder variable is defined, it explicitly indicates which recorder
+class should be used and for which file defined by ScanFile (based on the 
+order).
+
+Example 1:
+
+::
+
+    ScanFile = myexperiment.spec
+    ScanRecorder = FIO_FileRecorder
+
+    FIO_FileRecorder will write myexperiment.spec file.
+
+Example 2:
+
+::
+
+    ScanFile = myexperiment.spec, myexperiment.h5
+    ScanRecorder = FIO_FileRecorder
+
+    FIO_FileRecorder will write myexperiment.spec file and
+    NXscan_FileRecorder will write the myexpriment.h5. The selection of the
+    second recorder is based on the extension.
+
+.. seealso:: More about the extension to recorder map in
+             :ref:`sardana-writing-recorders`.
+
+.. _sharedmemory:
+
+SharedMemory
+~~~~~~~~~~~~
+*Not mandatory, set by user*
+
+Its value is of string type and it indicates which shared memory recorder should
+be used during the scan e.g. "sps" will use SPSRecorder (sps Python module
+must be installed on the PC where the MacroServer runs).
+
+.. seealso:: For more information about the implementation details of the scan
+             macros in Sardana, see 
+             :ref:`scan framework <sardana-macros-scanframework>`
+
+.. _sourceinfo:
+
+SourceInfo
+~~~~~~~~~~
+*Not mandatory, set by user*
+
```

Comment:
Missing documentation

Reply:
Same as before ...

Reply:
Fixed. The explanation is not the most explicit one, but could be enough at this moment.